docs/product/product_strategy.md

```diff
@@ -22,3 +22,27 @@ CISA lacks a scalable, efficient, and secure method of managing the .gov TLD pro
 | **Data:** The program maintains authoritative contacts at, metadata about, and hostname information for all registered .gov domains, and is able to track that .gov domains are actually used | - Time-to-generate internal reports decreases <br /> - Results of periodic data quality audit show improvements month-over-month                                                                              |
 | **User satisfaction:** Getting a .gov domain is as easy and intuitive as possible                                                                                                             | - Completion rate of form improves <br /> - Time from domain request to approval decreases <br /> - Number of domains requiring analyst data changes decreases                                                        |
 | **Program reputation and experience:** The .gov program is viewed as trustworthy and responsive                                                                                               | - Response time for inquiries decreases <br /> - Resolution time decreases <br /> - Rate of repeat issues for tickets decreases <br /> - Number of SLTT organizations in Community of Practice increases                                      |
+
+## Bug triage
+
+Bugs on production software need to be documented quickly and triaged to determine if fixes need to be made outside of the normal release cadence. Triage levels will be Critical, High, Medium, and Low to indicate the level of priority for fix, not neccessarily the level of severity. See below for more details
+
+**Critical**- should only be determined by the product owner and means the fix for this critical bug needs to have a quick fix for it created ASAP. This is the only case where a bug fix can be added outside of the normal release cycle and directly onto the stable release.
+**High**- Can be determined by product owner or other team member, and indicates this bug is critical enough to warrant being added into the current sprint.
+**Medium**- Should be added to a sprint coming up but is not blocking users, or enough users to warrant rushing it into a sprint
+**Low**- A minor bug, that could even wait until after the next big launch date to be implemented.
+
+### Steps for Triaging
+
+1. When a bug is found, whether by a developer/designer or from feedback from an end user, a ticket should be made immediately. The actual maker of the ticket can be a member of the product team as needed.
+2. This bug ticket immediately gets a priority added Critical/High/Medium/Low, with Critcal requiring the product owner's request.
```

Comment:
* typo at `Critcal`
* consider changing "product owner's request" to "product owner's consent"